addons/editor-buttons-reverse-order/addon.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Reverse order of project controls",
+  "description": "Moves the green flag and stop buttons to the right and the full screen button to the left, as in Scratch 2.0",
+  "tags": ["editor", "projectPlayer"],
```

Comment:
I think we should add the featured tag. Other 2.0-like addons have it (columns, editor-stage-left, etc.)